moveit_ros/planning_interface/move_group_interface/src/demo.cpp

```diff
@@ -68,7 +68,7 @@ void demoPick(moveit::planning_interface::MoveGroupInterface& group)
 
     grasps.push_back(g);
   }
-  group.pick("bubu", grasps);
+  group.pick("bubu", grasps, false);
```

Comment:
for readability can you make this a const?

    const bool plan_only = false;
    (..., plan_only)

Reply:
I had actually meant to just change this line (and other similar lines) to not have a hidden false value but instead make it a const right before the call:

    const bool plan_only = false;
    group.pick("bubu", grasps, plan_only);

Reply:
But the approach you used of having default values in the function headers i actually like better, except please remove the ``const``